main/main.go

```diff
@@ -216,6 +225,33 @@ func main() {
 
 	// Create listeners for each of the subnets configured as a source
```

Comment:
Looks like this comment is duplicated on line 254. Was that intended?

Reply:
I moved it, but I think the original crept back in as a merge conflict. Thanks for pointing it out. Addressed in https://github.com/ava-labs/awm-relayer/pull/344/commits/5c6422e8977a26bfbae4c62abc2678225547e3ae